security/src/main/java/uk/ac/ox/ndph/mts/security/authorisation/AuthorisationService.java

```diff
@@ -90,56 +69,56 @@ public boolean authorise(String requiredPermission) {
 
     /**
      * Authorise request with list of sites
-     * @param requiredPermission the required permission
-     * @param entitiesSiteIds list of site ids to check if the user is authorised on them
+     * @param requiredPerm the required permission
+     * @param siteIds list of site ids to check if the user is authorised on them
      * @return true if authorised - has the required permission and is authorised on all sites in entitiesSiteIds
      */
-    public boolean authorise(String requiredPermission, List<String> entitiesSiteIds)  {
-
-
+    public boolean authorise(String requiredPerm, List<String> siteIds)  {
```

Comment:
Nothing to do with your refactor Darren just an extra thought.   This method makes GET requests to practitioner service, site service and role service, to get the data it needs to do this logic.   I think it's unavoidable (leaving aside caching for now).    But it's a bit hidden to the uninitiated.  Would it be useful to future developers to add a big comment, flagging that this method does all this traffic?